### PR TITLE
Create separate folder for pojos

### PR DESCRIPTION
**Pull request by holounic**
Create separate folder for pojos (Builder, GitHubData, Log, etc). I think it's more logical to have them in a separate folder because these classes don't belong only to the storage part of the app. Instances of them are created in GitHub and BuildBot fetchers. Stage of another [PR](https://github.com/googleinterns/step240-2020/pull/91)

Change access modifiers from package-private to public to make POJOs accessible from other packages (fetchers, storage).

### CODE REVIEW

File: src/main/java/com/google/graphgeckos/dashboard/datatypes/Builder.java [hunk 1]
```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.graphgeckos.dashboard.storage;
+package com.google.graphgeckos.dashboard.datatypes;
 
 import java.util.ArrayList;
 import java.util.List;
+
```
Comment (krasimir-google) [inline src/main/java/com/google/graphgeckos/dashboard/datatypes/Builder.java:19]: please keep all (non-static) imports in a single block: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

File: src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java [hunk 2]
```diff
@@ -18,15 +18,13 @@
...
 import com.google.cloud.datastore.Query;
 import com.google.cloud.datastore.StructuredQuery.OrderBy;
-import com.google.cloud.datastore.StructuredQuery.PropertyFilter;
-import com.google.cloud.datastore.TimestampValue;
-import com.google.cloud.Timestamp;
 import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Date;
 import java.util.List;
+import com.google.graphgeckos.dashboard.datatypes.BuildInfo;
```
Comment (krasimir-google) [inline src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java:23]: please keep the imports in ASCII sort order (move this one and following to after line 20):
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

File: src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java [hunk 3]
```diff
@@ -20,6 +20,7 @@
 import java.util.ArrayList;
 import java.util.concurrent.TimeoutException;
 import java.util.List;
+import com.google.graphgeckos.dashboard.datatypes.*;
```
Comment (krasimir-google) [inline src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java:23]: please don't use wildcard imports: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports
(and keep these in sorted order)

File: src/main/java/com/google/graphgeckos/dashboard/datatypes/Builder.java (hunk 1 above)
Comment (holounic) [reply src/main/java/com/google/graphgeckos/dashboard/datatypes/Builder.java]: Done!

File: src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java (hunk 2 above)
Comment (holounic) [reply src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java:21]: Done! Sorry, IDEA does this automatically, I didn't notice it

File: src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java (hunk 3 above)
Comment (holounic) [reply src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java]: Done! Sorry, IDEA does this automatically, I didn't notice it

File: src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java [hunk 4]
```diff
@@ -20,6 +20,11 @@
 import java.util.ArrayList;
 import java.util.concurrent.TimeoutException;
 import java.util.List;
+import com.google.graphgeckos.dashboard.datatypes.Builder;
```
Comment (krasimir-google) [inline src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java:23]: please keep the imports in ASCII sort order (move this one and following to after line 20):
(move this and following lines to after line 18)

**Review by krasimir-google — APPROVED** [review]
Looks good after one last place of reordering imports

File: src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java (hunk 4 above)
Comment (holounic) [reply src/test/java/com/google/graphgeckos/dashboard/storage/DatastoreRepositoryTests.java:19]: Done!

**Review by MagoDelBlocco — APPROVED** [review]

**Pull request merged**